metamorph/src/main/resources/schemata/metamorph.xsd

```diff
@@ -776,7 +807,7 @@
 
     <element name="lookup">
         <annotation>
-            <documentation>Performs a table lookup</documentation>
+            <documentation>Performs a table lookup. When multiple keys match, lookup selects the last matching key.</documentation>
```

Comment:
Hm, that's not necessarily true. It depends on the map implementation. `lookup()` only ever sees a single value.

Reply:
I would revert my suggestion.then. But I do not know where to add it. Should this be documented with all map implementations?

Reply:
> Should this be documented with all map implementations?

Probably, yes. And I would suggest that it should be a separate pull request.

Reply:
+1